### ISSUE TITLE
Doesn't update after changes

### ISSUE THREAD

**Issue by mindplay-dk** [SanderRonde/phpstan-vscode#67]
Given:

- you have a file `A` that defines a class `A`.
- and you have a file `B` that calls a method on an instance of class `A`.
- an inspection error is currently showing for the call in file `B`.

Now, if you go and correct the type declarations in file `A`, the error in file `B` still shows.

My workaround is I go to file `B`, insert a space character and press CTRL+S to save, then remove the space character and save again - this triggers the expected update, but it's kind of clunky to work like this.

I'm using your extension release `v2.2.26` in VS Code version:

```
Version: 1.88.0 (user setup)
Commit: 5c3e652f63e798a5ac2f31ffd0d863669328dc4c
Date: 2024-04-03T13:26:18.741Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22631
```

**Comment by SanderRonde** [comment]
This should be fixed in the pre-release version. Can you check if that's indeed the case?

**Comment by mindplay-dk** [comment]
Did you release this change already? Because now I can't recreate the issue anymore. 😄

**Comment by SanderRonde** [comment]
I did not, you can switch to the pre-release version and back in the extension view. Maybe you're just on the pre-release version? :)

If you click on the extension in the extension view I'm pretty sure it'll show you the version. What version does it show now? (and does it show a big green pre-release tag?)

**Comment by mindplay-dk** [comment]
No, it's `v2.2.26` still.

Maybe it only happens with certain kinds of changes? Not sure. It's not happening now. 🤷‍♂️

**Comment by SanderRonde** [comment]
Hmm strange. I'll close this issue though, since it's probably fixed in the pre-release version. Let me know if you run into it again in that version :)
